src/app/services/xml-parsers/named-entities-parser.service.ts

```diff
@@ -0,0 +1,343 @@
+      });
+    }
+
+    return { lists: parsedLists, entities: parsedEntities, relations: parsedRelations };
+  }
+
+  private parseList(list: XMLElement) {
+    const parsedList: NamedEntitiesList = {
+      type: GenericElementComponent, // TODO: Set NamedEntitiesListComponent
+      id: list.getAttribute('xml:id') || xpath(list),
+      label: '',
+      namedEntityType: this.getListType(list.tagName),
+      content: [],
+      sublists: [],
+      originalEncoding: list,
+      relations: [],
+      description: [],
+      attributes: this.parseAttributes(list),
+    };
+    list.childNodes.forEach((child: XMLElement) => {
+      if (child.nodeType === 1) {
+        switch (child.tagName.toLowerCase()) {
+          case 'head':
+            parsedList.label = this.genericParserService.parse(child);
+            break;
+          case 'desc':
+            parsedList.description.push(this.genericParserService.parse(child));
+            break;
+          case 'relation':
+            parsedList.relations.push(this.parseRelation(child));
+            break;
+          case 'listrelation':
+            child.querySelectorAll<XMLElement>('relation').forEach(r => parsedList.relations.push(this.parseRelation(r)));
+            break;
+          default:
+            const listsToParse = this.getListsToParseTagNames();
+            if (listsToParse && listsToParse.indexOf(child.tagName) >= 0) {
+              const parsedSubList = this.parseList(child);
+              parsedSubList.id = parsedSubList.id;
+              parsedList.sublists.push(parsedSubList);
+              parsedList.content = parsedList.content.concat(parsedSubList.content);
+              parsedList.relations = parsedList.relations.concat(parsedSubList.relations);
+            } else {
+              parsedList.content.push(this.parseNamedEntity(child));
+            }
+        }
+      }
+    });
+    if (!parsedList.label && list.getAttribute('type')) {
+      parsedList.label = list.getAttribute('type');
+    }
+
+    return parsedList;
+  }
+
+  private parseNamedEntity(xml: XMLElement): NamedEntity {
+    switch (xml.tagName) {
+      case 'person':
+        return this.parsePerson(xml);
+      case 'personGrp':
+        return this.parsePersonGroup(xml);
+      case 'place':
+        return this.parsePlace(xml);
+      case 'org':
+        return this.parseOrganization(xml);
+      case 'event':
+        return this.parseEvent(xml);
+      default:
+        console.log('Warning! Parser not implemented for this element in list', xml);
+
+        return;
+    }
+  }
+
+  private parseGenericEntity(xml: XMLElement): NamedEntity {
+    const elId = xml.getAttribute('xml:id') || xpath(xml);
+    const entity: NamedEntity = {
+      type: GenericElementComponent, // TODO: Set NamedEntitiesComponent
+      id: elId,
+      originalEncoding: xml,
+      label: this.genericParserService.parse(xml),
+      namedEntityType: this.getEntityType(xml.tagName),
+      content: [],
```

Comment:
`content: Array.from(xml.childNodes).map((subchild: XMLElement) => this.parseEntityInfo(subchild))`

and remove line 178